### ISSUE TITLE
error messages

### ISSUE THREAD

**Issue by ahahahaah** [Adrianotiger/desktopPet#124]
hello 
ive downladed the portable version, when i right clic on the icon of the taskbar i have this error message
![image](https://github.com/Adrianotiger/desktopPet/assets/69671446/dbead022-f1c0-4426-8a49-165d79a57a92)

so from the same icon i choosed the installation option
now when i launch the application it doesnt launch at all
![image](https://github.com/Adrianotiger/desktopPet/assets/69671446/26e48c03-6921-49c2-b4f0-6fc2897b766c)

3rd problem i remember on the initial screenmate poo whith a right clic on it we could choose all the actions he can do
in this github version there is no right clic menu to choose the different actions.

(i use windows 7 64bits)

**Comment by Adrianotiger** [comment]
Hi,
The application opens this page to retrieve the pet list:
[https://raw.githubusercontent.com/Adrianotiger/desktopPet/master/Pets/pets.json](https://raw.githubusercontent.com/Adrianotiger/desktopPet/master/Pets/pets.json)

Could it be that Windows 7 doesn't have the certificate anymore? Can you navigate to this page with Internet Explorer without errors?

For the 3rd problem, you just need to press SHIFT when you execute the application to enter in the "debug" mode.

**Comment by ahahahaah** [comment]
ive tested your link and it display a page with this code text only:
`{
  "pets" : [
    {
      "folder" : "ssj-goku",
      "author" : "RedSparr0w",
      "lastupdate" : "2019-03-04"
    },
    {
      "folder" : "esheep64",
      "author" : "Adriano",
      "lastupdate" : "2019-02-21"
    },
    {
      "folder" : "bbunny",
      "author" : "Adriano",
      "lastupdate" : "2019-02-25"
    },
    {
      "folder" : "neko",
      "author" : "Adriano",
      "lastupdate" : "2019-06-28"
    },
    {
      "folder" : "pingus",
      "author" : "Adriano",
      "lastupdate" : "2019-07-08"
    },
    {
      "folder" : "negima",
      "author" : "Adriano",
      "lastupdate" : "2019-10-16"
    },
    {
      "folder" : "red_sheep",
      "author" : "Oliver B.",
      "lastupdate" : "2021-05-02"
    },
    {
      "folder" : "orange_sheep",
      "author" : "Oliver B.",
      "lastupdate" : "2021-05-02"
    },
    {
      "folder" : "yellow_sheep",
      "author" : "Oliver B.",
      "lastupdate" : "2021-05-02"
    },
    {
      "folder" : "green_sheep",
      "author" : "Oliver B.",
      "lastupdate" : "2021-05-02"
    },
    {
      "folder" : "blue_sheep",
      "author" : "Oliver B.",
      "lastupdate" : "2021-05-02"
    },
    {
      "folder" : "purple_sheep",
      "author" : "Oliver B.",
      "lastupdate" : "2021-05-02"
    },
    {
      "folder" : "pink_sheep",
      "author" : "Oliver B.",
      "lastupdate" : "2021-05-02"
    },
    {
      "folder" : "fox",
      "author" : "Michelle!",
      "lastupdate" : "2022-08-03"
    },
    {
      "folder" : "mimiko",
      "author" : "Madnuttah",
      "lastupdate" : "2022-08-25"
    }
  ]
}

`
didn't know for the shift method thanks

last problem there is no sound even if i set the sound to 10 and allow it

i have an old release (the 1.2.5) and it displays without error message the online pets tab and it run sounds without problem too.
you could tell me to continue to use it then but it doesnt have the hd size so they are small on the screen^^


thanks

**Comment by ahahahaah** [comment]
it seems that the green version (the 1.2.5 i have) has sound and no bug to display the online pets tab
so i wanted to downlad the news green version but i don't find any link for that

**Comment by Adrianotiger** [comment]
Yes, it is a list of all available pets. I was just wondering if you got any SSL-certificate errors on your browser or if you get the "green" secure server connection. It is important that you test it with IE or Chrome, as Firefox can download new certificates and the others are using the windows certificate (like the application).

The original pet doesn't have any sound. You need to download one of the other mates to hear a sound some times.

The old release doesn't get the pets from github, but from my old page (esheep.petrucci.ch). This page can be accessed via http or https.

**Comment by ahahahaah** [comment]
the result of the page i added in my message is what i have with IE (IE 11 because i use windows 7) 
if there is a new green version of the sheep, i dont find any link to get it. it is not available anymore?

**Comment by Adrianotiger** [comment]
Yes, it is available on the old page and is available also on the github.
You can find all pets on this folder:
https://github.com/Adrianotiger/desktopPet/tree/master/Pets

What you can do is download the xml file from:
https://github.com/Adrianotiger/desktopPet/tree/master/Pets/green_sheep

and drag it over your current pet. This will replace the original one with the new one manually.

**Comment by ahahahaah** [comment]
ok so i have downladed the **animations.xml** file and dragged it over the **desktoppet.exe**
the 1st time it only launched the purple sheep (so it didn't become green)
now when i do it again or when i launch only desktoppet.exe i have this window
![image](https://github.com/Adrianotiger/desktopPet/assets/69671446/4e3226f8-fb2a-47da-961c-94411f6ad9a6)
and when i clic ok
![image](https://github.com/Adrianotiger/desktopPet/assets/69671446/6c0bb5c4-d340-435a-928a-db164fbfb725)

**Comment by ahahahaah** [comment]
ok i deleted the config file, now i can relaunch the program
but still cant change it to green

**Comment by ahahahaah** [comment]
maybe the problem is because on the pets page i can see "This folder contains all pets for the Windows 10 application and WebSheep script."
it is windows 10 version.
the xml are not compatible with windows 7 maybe

**Comment by ahahahaah** [comment]
this message to notice that i finally succed: i have copyed the desktop.config file from the 1.2.5 green sheep to the 1.3.1 version which replace the desktop.config file of the 1.3.1 sheep.

**but** it remains the problem of the online pets tab. 
why it is possible to have it with the 1.2.5 green sheep and not with the 1.3.1. version
the certificate works for one and not the other?

**Comment by Adrianotiger** [comment]
I don't know what it could be. I bet on the root certificate for the https address.
But also the version 1.2.5 uses the new GiHub address. So I really don't know why the xml is not loaded on Windows 7. 

And the 1.2.6 version? I updated some libraries on the 1.3.1 version, but they should work also on windows 7.

**Comment by ahahahaah** [comment]
i could give it a try but all links lead to the last version
can you show me how to access to the v1.2.6 ? (portable version)

**Comment by ahahahaah** [comment]
ok found, i had to develop the "asset" section and downlad the .gz file

in the version 1.2.6, i can also have the online pets tab normally with all pets diplayed and available. 
so it seems that something has changed from this version and causes the error with 1.3.1 (in my case at any rate)

**Comment by Fleximaal** [comment]
UNSUBSCRIBE

 

Van: Adriano ***@***.***> 
Verzonden: woensdag 6 september 2023 16:56
Aan: Adrianotiger/desktopPet ***@***.***>
CC: Subscribed ***@***.***>
Onderwerp: Re: [Adrianotiger/desktopPet] error messages (Issue #124)

 

I don't know what it could be. I bet on the root certificate for the https address.
But also the version 1.2.5 uses the new GiHub address. So I really don't know why the xml is not loaded on Windows 7.

And the 1.2.6 version? I updated some libraries on the 1.3.1 version, but they should work also on windows 7.
